src/commands/utility/randomuser.ts

```diff
@@ -2,22 +2,19 @@ import Command from '../../lib/structures/Command';
 import { TypicalGuildMessage } from '../../lib/types/typicalbot';
 import { Modes } from '../../lib/utils/constants';
 
-const regex = /(-o(?:nline)?\s)?/i;
-
 export default class extends Command {
     aliases = ['ruser'];
     mode = Modes.LITE;
 
     execute(message: TypicalGuildMessage, parameters: string) {
-        const args = regex.exec(parameters);
         const role = parameters
             ? message.guild.roles.cache.get(parameters)
                 || message.guild.roles.cache.find((role) => role.name.toLowerCase() === parameters.toLowerCase())
             : message.mentions.roles.first();
 
         const members = role
             ? role.members.filter((m) => !m.user.bot)
-            : args
+            : ['-o', '-online'].includes(parameters)
```

Comment:
What about `-o` and `--online`? Just a thought.

Reply:
-o will work. Do you want to add --online as an acceptable answer? What are the acceptable args you want to accept for this?

Reply:
no, `-o` and `-online` are the only acceptable args - according to the regex

Reply:
I know. I'm purposing to change it.

Reply:
Should be a separate issue/pr

Reply:
A separate PR would make this one redundant.

Reply:
`-r` and `-role` would not be a fix.

Reply:
Can be changed.

Reply:
Note the role argument requires a role being provided. If it's just -r or -role i don't think that adds anything beneficial or did i misunderstand the request

Reply:
To be clear, I don't like `-r` and `-role`. I prefer the CLI standard of one dash for shorthand, two dashes for longhand.

With regard to `-r`, I'm saying that would prefer the role argument (e.g. -r VIP), a CLI type of output.

I suppose this could be part of a greater effort to make arguments more consistent.

Reply:
TBH i'm confused why any flags like this are even wanted. Are you sure about this? CLI standards are built with developers in mind. A bot isn't made for devs but average end-users. I'm not sure if this would be a helpful change to user experience but a harmful one.

99% of users are used to do doing
`$mute @user reason`
Having them do 
`$mute --user=@user --reason="my reason"` 
Seems worse to me

Reply:
I'm okay with that, I just want to see it be consistent with everything else.